h/accounts/views.py

```diff
@@ -141,6 +141,11 @@ def _logout(self):
                renderer='json')
 class AjaxAuthController(AuthController):
 
+    def __init__(self, request):
```

Comment:
This is added so that `AjaxAuthController` doesn't do a bunch of setup (from the `AuthController` constructor) that is not necessary. It means we don't need to register routes for `TestAjaxAuthController`.